A few no-more-404s graduation_report HTML errors via W3's HTML validator

Seeing a few HTML validation errors via https://validator.w3.org/nu/#textarea for the no-more-404s.yaml `graduation_report` HTML:

1. **Error:** Bad value 100% for attribute `width` on element `img`: Expected a digit but saw `%` instead.
   From line 21, column 3; to line 21, column 111
   ```html
   figure>↩
   <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" width="100%" height="auto" />↩ 
   <fi
   ```

1. **Error:** Bad value `auto` for attribute `height` on element `img`: Expected a digit but saw `a` instead.
   From line 21, column 3; to line 21, column 111
   ```html
   figure>↩
   <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" width="100%" height="auto" />↩ 
   <fi
   ```

1. **Error:** Bad value `100%` for attribute `width` on element `img`: Expected a digit but saw `%` instead.
    From line 27, column 3; to line 27, column 104
    ```html
    figure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-001.png" width="100%" height="auto" />↩
    <fi
    ```

1. **Error:** Bad value `auto` for attribute `height` on element `img`: Expected a digit but saw `a` instead.
    From line 27, column 3; to line 27, column 104
    ```html
    figure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-001.png" width="100%" height="auto" />↩
    <fi
    ```

1. **Error:** Bad value 100% for attribute width on element img: Expected a digit but saw % instead.
    From line 32, column 5; to line 32, column 103
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/car.gif" width="100%" height="auto" />↩
    <
    ```

1. **Error:** Bad value auto for attribute height on element img: Expected a digit but saw a instead.
    From line 32, column 5; to line 32, column 103
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/car.gif" width="100%" height="auto" />↩
    <
    ```

1. **Error:** Bad value 100% for attribute width on element img: Expected a digit but saw % instead.
    From line 38, column 5; to line 38, column 106
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-002.png" width="100%" height="auto" />↩
    <
    ```

1. **Error:** Bad value auto for attribute height on element img: Expected a digit but saw a instead.
    From line 38, column 5; to line 38, column 106
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-002.png" width="100%" height="auto" />↩
    <
    ```

1. **Error:** Bad value 100% for attribute width on element img: Expected a digit but saw % instead.
    From line 43, column 5; to line 43, column 106
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-003.png" width="100%" height="auto" />↩
    <
    ```

1. **Error:** Bad value auto for attribute height on element img: Expected a digit but saw a instead.
    From line 43, column 5; to line 43, column 106
    ```html
    gure>↩
    <img src="/static/images/experiments/no-more-404s/graduation/gr-003.png" width="100%" height="auto" />↩
    <
    ```

Similarly, the **eslint-plugin-jsx-a11y** module noted there was an unclosed `<p>` tag, and subsequently complained about a handful of `<img>` tags that were lacking `alt=""` attributes:

```
> jsx-a11y-test@1.0.0 test /Users/pdehaan/dev/tmp/del/jsx-a11y-test
> eslint .


/Users/pdehaan/dev/tmp/del/jsx-a11y-test/index.js
   8:3  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt
  13:3  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt
  19:3  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt
  24:5  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt
  30:5  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt
  35:5  error  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/img-has-alt

✖ 6 problems (6 errors, 0 warnings)
```

--- a/content-src/experiments/no-more-404s.yaml
+++ b/content-src/experiments/no-more-404s.yaml
@@ -85,42 +85,42 @@ locale_blocklist:
 order: 6
 graduation_report: >
   <h3>Test Pilot No More 404s Graduation Report</h3>
-  <p>Last winter, some folks from the Test Pilot team got together with some folks from the <a href="https://archive.org/" target="_blank">Internet Archive</a> and hatched a plan. On the Test Pilot side of things, we were busy building our platform and getting experiments out into the wild. Meanwhile, the team at the Internet Archive was prototyping an add-on to help users avoid dead ends on the Web by checking if they had archived versions of sites available in the <a href="https://archive.org/web/" target="_blank">Wayback Machine</a> for users who encountered 404 errors.<p>
+  <p>Last winter, some folks from the Test Pilot team got together with some folks from the <a href="https://archive.org/" target="_blank">Internet Archive</a> and hatched a plan. On the Test Pilot side of things, we were busy building our platform and getting experiments out into the wild. Meanwhile, the team at the Internet Archive was prototyping an add-on to help users avoid dead ends on the Web by checking if they had archived versions of sites available in the <a href="https://archive.org/web/" target="_blank">Wayback Machine</a> for users who encountered 404 errors.</p>
   <p>We launched the No More 404s experiment with a few goals in mind. We know from user research that 404 pages are a persistent nuisance on the Web, and a No More 404s add-on seemed like a really sweet utility to help steer users toward their intended content. Additionally, the Test Pilot team wanted to find out how a partner organization like the Internet Archive could help bring testable features into Firefox.</p>
   <p><strong>Here’s how it went down</strong></p>
   <p>When we launched No More 404s, the user interface was simple and unobtrusive. If a participant visited a 404 on the Web – specifically if a site that returned a HTTP header code 404 – the No More 404s experiment would send a request to the Wayback Machine to see if the service had an archived version of the page. If one existed, No More 404s would drop a notification bar on the page with a link to it. Users could then visit the content archived in the Wayback Machine, dismiss the notification bar, or just continue browsing as usual.</p>
   <figure>
-  <img src="/static/images/experiments/no-more-404s/experiments_experimentdetail/2/4/24ddd4335aca6b96cca9106a6f3411d2_image_1470245154_0440.jpg">
-  <figcaption>We launched No More 404s with a restrained design.</figcaption>
+    <img src="/static/images/experiments/no-more-404s/experiments_experimentdetail/2/4/24ddd4335aca6b96cca9106a6f3411d2_image_1470245154_0440.jpg">
+    <figcaption>We launched No More 404s with a restrained design.</figcaption>
   </figure>
   <p>Later on in the experiment we made a few changes to the add-on. First, we added in more common error codes associated with missing Web assets. Then we made a much larger change when we asked our design team to increase the visibility of the add-on. We wanted to see how these changes would affect user perception of the experiment.</p>
   <figure>
-  <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" width="100%" height="auto" />
-  <figcaption>In December we changed to something a bit crazier.</figcaption>
+    <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" />
+    <figcaption>In December we changed to something a bit crazier.</figcaption>
   </figure>
-  <p><strong>Here's what we learned</strong></p>
+  <p><strong>Here’s what we learned</strong></p>
   <p>Both teams wanted to get a sense of how many of the URLs that display 404 errors today had pages archived in the Wayback Machine. <a href="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/edit#gid=287298530" target="_blank">We tracked total 404s</a> encountered by Test Pilot participants (1,275,165) as well as results returned by the Archive (185,081). Over the course of the experiment, we averaged a result 14.5 percent of the time, but also observed that over the course of the experiment the daily average rose.</p>
   <figure>
-  <img src="/static/images/experiments/no-more-404s/graduation/gr-001.png" width="100%" height="auto" />
-  <figcaption>Both the total 404s encountered and the percentage of returned codes grew over the course of the experiment.</figcaption>  
+    <img src="/static/images/experiments/no-more-404s/graduation/gr-001.png" />
+    <figcaption>Both the total 404s encountered and the percentage of returned codes grew over the course of the experiment.</figcaption>  
   </figure>
   <p>In addition to 404s, which are common on the Web, there are other error codes users encounter. For example, users may reach a page with a 408 error code when a server takes too long to respond. In early November, the Internet Archive team decided to <a href="https://github.com/internetarchive/FirefoxNoMore404s/blob/master/src/scripts/background.js#L28" target="_blank">add more error codes to the experiment</a>. We hypothesized that adding additional error codes would cause the overall percentage of sites returned by the Wayback Machine to drop relative to total queries made by the experiment. However, the opposite was true: in the graph above there is a modest uptick in the percentage of valid returns in early November after we made the change.</p>
   <figure>
-    <img src="/static/images/experiments/no-more-404s/graduation/car.gif" width="100%" height="auto" />
+    <img src="/static/images/experiments/no-more-404s/graduation/car.gif" />
     <figcaption>Just look at this guy!</figcaption>
   </figure>
   <p>In early December we updated the user interface for the experiment and featured an adorable animated GIF. Changing the user interface affected how participants interacted with the No More 404s experiment. Prior to the changes, the experiment was tucked out of the way, so we wanted to see what happened if we made it more flashy. We assumed users might dismiss the UI more frequently than before, and they did, but we were surprised to see that the rate at which users dismissed the pop-up increased sixfold.</p>
   <p>We also observed an uptick in the percentage of returned results that correlates to these UI changes. Of course, correlation is not causation, and this uptick remains a mystery: changing UI should not have affected the relative rate of successful returns from the Wayback Machine.</p>
   <figure>
-    <img src="/static/images/experiments/no-more-404s/graduation/gr-002.png" width="100%" height="auto" />
+    <img src="/static/images/experiments/no-more-404s/graduation/gr-002.png" />
     <figcaption>The rate at which users ignored the interface shot up after we changed the user interface.</figcaption>  
   </figure>
   <p>We also wanted to know how often experimenters encountered 404 errors as they browse. While this experiment did not track total sites visited by our users (because of privacy concerns), we did keep tabs on how many times each browser hit a 404 each week. Throughout the life of the experiment, roughly 50 percent of participants encountered at least one 404 per week. Of those who encountered 404s, well over 25 percent did so at a rate of four or more a week!</p>
   <figure>
-    <img src="/static/images/experiments/no-more-404s/graduation/gr-003.png" width="100%" height="auto" />
+    <img src="/static/images/experiments/no-more-404s/graduation/gr-003.png" />
     <figcaption> Breakdown of the number of 404s encountered by users each week.</figcaption>
   </figure>
-  <p><strong>Here's what happens next</strong></p>
+  <p><strong>Here’s what happens next</strong></p>
   <p>While we’re graduating No More 404s from Test Pilot, and will be moving the add-on to the <a href="https://addons.mozilla.org/firefox/addon/wayback-machine_new/" target="_blank">Mozilla Add-ons</a> store so it’s accessible to all Firefox users. We won’t remove it from users who have it installed. If you currently have the add-on installed, you don’t need to do anything. The add-on will continue to update as usual. The Internet Archive team will also be adding some new features so users can add content to the Wayback Machine directly through the add-on! On the Firefox side, we will promote No More 404s on the homepage of the browser.</p>
   <p>Once the add-on is in its permanent home, our teams will continue to work together on studies that answer key questions, including:</p>
   <ul>

